tests/post_training/test_templates/helpers.py

```diff
@@ -14,10 +14,12 @@
 import numpy as np
 import torch
 from torch import nn
+from torch.nn import functional as F
 
 from nncf import Dataset
 from tests.torch.helpers import create_bn
 from tests.torch.helpers import create_conv
+from tests.torch.helpers import set_torch_seed
```

Comment:
let's not forget to discuss how can we avoid torch in common tests, this will be the problem when TF backend will be implementing